### ISSUE TITLE
VariablePlugin README

### ISSUE THREAD

**Issue by irmerk** [accordproject/cicero-ui#146]
# VariablePlugin

This should have developer documentation associated with it to showcase how it can be used and its purpose.

> VariablePlugin: A custom Slate plugin for using editable, highlighted variables within a clause

**Comment by tingaloo** [comment]
I'll give this one a shot!

**Comment by tingaloo** [comment]
I am trying to run the demo app on the master branch, but i get these error messages.
```
webpack:///./node_mo…olbar/index.js?:104 Uncaught TypeError: Cannot read property 'div' of undefined
    at eval (webpack:///./node_mo…olbar/index.js?:104)
    at Object../node_modules/@accordproject/markdown-editor/dist/FormattingToolbar/index.js (bundle.js:583)
    at __webpack_require__ (bundle.js:20)
    at eval (webpack:///./node_mo…Editor/index.js?:26)
    at Object../node_modules/@accordproject/markdown-editor/dist/SlateAsInputEditor/index.js (bundle.js:666)
    at __webpack_require__ (bundle.js:20)
    at eval (webpack:///./node_mo…r/dist/index.js?:49)
    at Object../node_modules/@accordproject/markdown-editor/dist/index.js (bundle.js:810)
    at __webpack_require__ (bundle.js:20)
    at eval (webpack:///./demo/src/index.js?:11)
```

Is there a version that has a more stable demo app?

**Comment by irmerk** [comment]
You'll want to develop `cicero-ui` within `template-studio-v2` with these instructions: https://github.com/accordproject/cicero-ui#development

**Comment by irmerk** [comment]
@tingaloo are you sure you're working off the latest version of master? The issue you're seeing was fixed in a [`markdown-editor` release](https://github.com/accordproject/markdown-editor/releases/tag/v0.5.16) and a [`cicero-ui` release](https://github.com/accordproject/cicero-ui/releases/tag/v0.0.34)

**Comment by tingaloo** [comment]
@irmerk doh! you're right. i can work off of this now, thanks!

**Comment by irmerk** [comment]
Opening this back up after 15 days stale.

**Comment by Innovator007** [comment]
Can i work on this?

**Comment by Innovator007** [comment]
Hi @irmerk can you review my pr [here](https://github.com/accordproject/cicero-ui/pull/172)

**Comment by irmerk** [comment]
Closed with #172
